CHANGELOG.rst

```diff
@@ -18,6 +18,10 @@ Unreleased
 * Switch from ``edx-sphinx-theme`` to ``sphinx-book-theme`` since the former is
   deprecated
 
+[3.62.8]
+--------
+feat: hooking enterprise enrollments up to platform signals to write unenrollment records
```

Comment:
docs nit: maybe worth mentioning here that we default this new field to null because it's really only applicable for enrollments written in some future, EMET-enabled reality.